contribs/gmf/apps/desktop/index.html

```diff
@@ -152,6 +192,7 @@
         module.constant('langUrls', langUrls);
         module.constant('authenticationBaseUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi');
         module.constant('fulltextsearchUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/fulltextsearch?limit=30&partitionlimit=5');
+        module.constant('gmfAltitudeUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/raster');
```

Comment:
It can have other attribute than the altitude, then `gmfRasterUrl` will be a better name, see http://sitn.ne.ch/theme/main ...